oauth-client/src/main/java/io/strimzi/kafka/oauth/client/JaasClientOauthLoginCallbackHandler.java

```diff
@@ -193,11 +215,20 @@ private long getHttpRetryPauseMillis(ClientConfig config, int retries) {
         return retryPauseMillis;
     }
 
+    private TokenProvider createProvider(final String token, final String tokenLocation) {
+        if (tokenLocation != null) {
```

Comment:
Do we want to accept the case where both parameters are non-null? It's ambiguous what the user's intent is in this case.

Reply:
I aded a logic that if both are set, the location (file) is used rather than the token, and warning is logged.